### PR TITLE
Add perfetto ui as a toggle

### PR DESCRIPTION
**Pull request by paulirish**
There's a lil toggle in the toolbar now 
image

It'll show perfetto UI and load the trace in it. And then you can toggle back

image

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -61,24 +62,65 @@ async function displayTrace(assetUrl, fileData) {
...
+    iframePerfetto.contentWindow.postMessage(payload, ORIGIN);
+    iframePerfetto.classList.add('traceloaded');
+  };
+
+  window.addEventListener('message', onPerfettoMsg);
+}
+
+function togglePerfettoViz(){
+  const iframePerfetto = $('iframe#ifr-perfetto');
+  const shouldShowPerfetto = iframePerfetto.classList.toggle('visible');
```
Comment (alexnj) [inline src/app.js:119]: Is this meant to write/toggle state here or `.contains` is the right use?

**Review by alexnj — APPROVED** [review]
🚢  + couple of nits.

File: src/app.js (hunk 1 above)
Comment (paulirish) [reply src/app.js:119]: good catch i'm relying on the return value of `toggle()` which.. i had to look up.. it's defined.  :)

image

kinda weird logically to read it. 
but.. beats a toggle followed by a contains?

File: src/app.js [hunk 2]
```diff
@@ -29,28 +29,29 @@ const devtoolsBaseUrl = `https://chrome-devtools-frontend.appspot.com/serve_rev/
...
- * @param {FullMetadata} fileData 
- * @returns 
+ * @param {string | undefined} assetUrl
+ * @param {FullMetadata} fileData
+ * @returns
  */
 async function displayTrace(assetUrl, fileData) {
   if (!assetUrl) {
     document.body.className = 'state--idle';
+    $('iframe#ifr-perfetto').className = '';
```
[the remark this answers is not in the document]
Comment (paulirish) [reply src/app.js]: fixed.

**Pull request merged**